game.py

```diff
@@ -0,0 +1,32 @@
+import random
+while(True):
+    n=random.randint(1,100)
+    i=1
+    while(True):
+        h=0
+        while(True):
+            h=input("hads bezan : ")
+            if h.isdigit():
+                h=int(h)
+                break
+            else:
+                print("dobare vard kon : ")

```

Comment:
خوبه که ورودی رو چک کردی

ولی خیلی ضروری نیست تو این تمرین
میتونی بکنیش یه فانکشن مجزا 
یا با try exceptبنویسیش 
اینطور
```suggestion
        while True:
            try:
                h=int(input("hads bezan : "))
            except:
                print("dobare vard kon : ")
            else:
                break
```

هر چند همینطور هم درسته